### PR TITLE
Addition of About me page

### PR DESCRIPTION
**Pull request by Ebot101**
This pr contains the code that solves Issue #2 by adding the webpage that displays the intro to each individual fellow as well as displays links to more in-depth information about each fellow.
Screenshot (261)

### CODE REVIEW

**Review by casseyshao — COMMENTED** [review]
This looks good to me! Code looks good! Later as a team if there is time, we can discuss data structure because I think all of us have overlapping data but that's a later minor thing!!:)

**Comment by Reggie-Amedee-Tech** [comment]
I love what you did with the nav bar. I also really like how the picture and name of the fellow looks. Everything looks aligned. 

Typically with typography, you want your headers, in this case, the name, and your sub headers, i'd image the about me, to be "half" in sizing. Meaning if you have a header thats like 50 px, you may want the subheader to be 25px. the content(about me information) can be be around 12.5 px. I feel like designing texts on a webpage in this way naturally guides users to where you want them to go, in this case, the name of the fellow, the about me sub header and the about me content.

I also feel like you can decrease the width of the about me paragraph? 

what do you guys think?

This looks amazing so far.

**Comment by Reggie-Amedee-Tech** [comment]
> Owner

That would be an interesting topic. I'd be down to chat about that.

File: app/__init__.py [hunk 1]
```diff
@@ -24,6 +24,36 @@
 ]
 
 
+pic_data = {
```
Comment (Pradyuman7) [inline app/__init__.py:27]: These constant kind of data (`pic_data`, and `about_me`) can live in other file to make the actual code file smaller and cleaner

File: app/static/styles/aboutMePage.css [hunk 2]
```diff
@@ -1,13 +1,138 @@
...
+.profile {
+    background-color: #1d539f !important;
+    text-align: center;
+    justify-content:center;
+    height: 250px;
+    display:flex;
+}
+
+.profile h1 {
+        font-size: 100px;
```
Comment (Pradyuman7) [inline app/static/styles/aboutMePage.css:61]: Better to use `em` for font sizes imo

**Pull request merged**